com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2020.1-to-2020.2.md

```diff
@@ -18,6 +18,14 @@ Currently, the only publicly accessible variables in the `ShaderVariablesGlobal`
 
 From Unity 2020.2, if you create a new [HDRP Asset](HDRP-Asset.md), the **MSAA Within Forward** Frame Setting is enabled by default.
 
+## Decal
+
+From Unity 2020.2, Decal no longer require a full Depth Prepass. Only Material with **Receive Decals** enable will be render during the Depth Prepass. Unless other option force it.
```

Comment:
From Unity 2020.2, decals no longer require a full Depth Prepass. HDRP only renders Materials with **Receive Decals** enabled during the Depth Prepass. Unless other options force it.